config/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/resource/OAuth2ResourceServerConfigurer.java

```diff
@@ -248,6 +274,35 @@ JwtDecoder getJwtDecoder() {
 		}
 	}
 
+	public class OpaqueConfigurer {
```

Comment:
I think we want something like OpaqueTokenConfigurer here. I realize we have JwtConfigurer, but Jwt has Token in the acronym.